src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
+@import "./utils/variables";
+@import "./blocks/calendar";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
```

Comment:
```suggestion

```
It's better to move your general styles to a separate file to keep your main file clean and clear